openedx/features/course_experience/tests/views/test_course_home.py

```diff
@@ -99,16 +102,23 @@ def test_updates_tool_visibility(self):
         response = self.client.get(url)
         self.assertContains(response, TEST_COURSE_UPDATES_TOOL, status_code=200)
 
-    @skip("This test verifies the query counts on Home Page. It is flaky. Will see after ginkgo release.")
     def test_queries(self):
         """
         Verify that the view's query count doesn't regress.
         """
+
+        #deletes course updates block to avoid extra call(s) to mongo db
+        updates_usage_key = get_course_info_usage_key(self.course, 'updates')
+        try:
+            modulestore().delete_item(updates_usage_key, self.user.id)
+        except ValueError:
+            pass
```

Comment:
can we add a comment here to indicated that we have to pick changes from openedx during hawthorn rebase?

Reply:
Done